data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,52 @@
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
+
+
+# Climate Change
+EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected population loss rate and is low income"
+EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected agriculture loss rate and is low income"
+EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected building loss rate and is low income"
+
+# Clean energy and efficiency
+PM25_LOW_INCOME = "PM2.5 exposure and is low income"
+ABOVE_90TH_FOR_COST_BURDEN_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for energy cost burden score and is low income"
```

Comment:
why the word "score" here? can you please make these match the names of the variables (in this case, the variable is just `energy burden`), thanks!

Reply:
@lucasmbrown-usds This is verbatim from the issue #921. 
![Screen Shot 2021-12-05 at 2 24 17 PM](https://user-images.githubusercontent.com/7760225/144760625-7cb9d253-fd8a-4f99-8cac-be2024c413c1.png)

Reply:
But, yes I will change this to match the variable